src/App.scss

```diff
@@ -107,3 +108,41 @@
 .excalidraw .RadioGroup__choice input {
   opacity: 0;
 }
+
+// Custom Context Menu
+.excalidraw .context-menu-item__label {
+  font-weight: normal !important;
+}
+
+.excalidraw .context-menu {
+  position: relative !important;
+  border-radius: var(--border-radius-large) !important;
+  box-shadow: none !important;
+  filter: drop-shadow(0 1px 10px var(--color-box-shadow)) !important;
+  list-style: none !important;
+  -webkit-user-select: none !important;
+  user-select: none !important;
+  margin: -0.25rem 0 0 0.125rem !important;
+  padding: 0.25rem 0.25rem !important;
+  background-color: var(--color-main-background) !important;
+  cursor: default !important;
+  border: none !important;
+}
+
+.excalidraw .context-menu-item-separator {
+  border-top: 0.5px solid #adb5bd !important;
+  margin-top: 0.25rem !important;
+  margin-bottom: 0.25rem !important;
```

Comment:
Ideally we'd use spacing values that are calculated from `--default-grid-baseline` https://docs.nextcloud.com/server/latest/developer_manual/html_css_design/css.html#element-structure-variables